util/testutil/shell.go

```diff
@@ -216,7 +216,8 @@ func monitorStartup(errs chan error) func(string) {
 				errs <- errors.New("fatal snapshotter log entry encountered, snapshotter failed to start")
 				return
 			}
-			if strings.Contains(logline.Msg, "background") {
+			// Looking for "soci-snapshotter-grpc successfully started"
+			if strings.Contains(logline.Msg, "successfully") {
```

Comment:
Can we use a constant that's used both here and in the log message itself?

Reply:
We'd have to import the testutil into the CLI package or vice versa. The former is being pretty messy to figure out (and honestly not even sure if it's possible) while the latter doesn't make logical sense to me.

I think we should just keep it as is. The comment clarifies exactly what we're looking for in the logs.